### PR TITLE
Upgraded the project to use assertJ 3.8.0

### PR DESCRIPTION
**Pull request by IAmWaseem**
Upgraded the project to use assertJ 3.8.0 - Removed broken test cases that give compile time error

### CODE REVIEW

File: kubernetes-assertions.iml [hunk 1]
```diff
@@ -13,6 +13,7 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" name="Maven: org.assertj:assertj-core:3.8.0" level="project" />
     <orderEntry type="library" name="Maven: io.fabric8:kubernetes-api:3.0.8" level="project" />
     <orderEntry type="library" name="Maven: javax.validation:validation-api:1.1.0.Final" level="project" />
     <orderEntry type="library" name="Maven: io.fabric8:openshift-client:3.0.3" level="project" />
```
Comment (rasheedamir) [inline kubernetes-assertions.iml:19]: do v really need this?

File: kubernetes-assertions.iml [hunk 2]
```diff
@@ -36,7 +37,7 @@
     <orderEntry type="library" name="Maven: dk.brics.automaton:automaton:1.11-8" level="project" />
     <orderEntry type="library" name="Maven: io.fabric8:fabric8-utils:3.0.8" level="project" />
```
Comment (rasheedamir) [inline kubernetes-assertions.iml:38]: do we need this?

File: kubernetes-assertions.iml (hunk 1 above)
Comment (IAmWaseem) [reply kubernetes-assertions.iml:19]: Its for intelliJ, I think we can remove this file entirely without any issue.

File: src/main/java/com/stakater/kubernetes/assertions/NavigationListAssert.java [hunk 3]
```diff
@@ -1,7 +1,6 @@
 package com.stakater.kubernetes.assertions;
 
-import org.assertj.core.api.AbstractAssert;
-import org.assertj.core.api.ListAssert;
+import org.assertj.core.api.*;
```
Comment (rasheedamir) [inline src/main/java/com/stakater/kubernetes/assertions/NavigationListAssert.java]: never use wildcard imports; disable settings inside IntelliJ

Comment (IAmWaseem) [reply src/main/java/com/stakater/kubernetes/assertions/NavigationListAssert.java]: I didn't know intelliJ was doing it, i'll disable it and add specific imports

**Comment by rasheedamir** [comment]
@IAmWaseem in the readme plz add reference to source and explain why we forked and now maintaining it ourselves

**Comment by IAmWaseem** [comment]
Okay i'll update the readme soon

**Review by rasheedamir — APPROVED** [review]
Looks good for now; lets proceed. Once we configure SonarQube then we can see the code coverage; but I am hoping its 100% covered and not compromised.

**Comment by IAmWaseem** [comment]
Alright, so should I merge it now?

**Comment by rasheedamir** [comment]
Yes

**Comment by rasheedamir** [comment]
Still not merged @IAmWaseem ?

**Pull request merged**